app/src/main/java/com/nervousfish/nervousfish/activities/CreateProfileHelper.java

```diff
@@ -40,7 +40,7 @@
     }
 
     /**
-     * Generates a KeyPair based on the type selected.
+     * Generates a list with one {@link KeyPair} based on the type selected.
```

Comment:
I understand you added it for that reason, but I don't see why we would need one method that returns a `List` with a single key and one that returns a `List` with multiple keys. Simply always return a `List` of keys, and the rest of the application should be able to handle that. imo.